packages/terra-graphs/docs/controls/PairedResult.md

```diff
@@ -167,6 +167,8 @@ Each paired result can have a style object in [Values](#values) level.
 
 -   `isCritical` toggle is disabled by default
 -   When `isCritical` toggle is enabled, an indicator will be shown surrounding the data point
+-   Providing invalid data to x or y will lead to console error.**
```

Comment:
is this meant to be bolded?

Reply:
fixed [here](https://github.com/cerner/terra-graphs/pull/24/commits/e52ebfc5e2040365cbd03027e1277864cbb4065a#diff-5afe4e3187fecc88b3a2658ae5a99f7157eb90c0f6f967a27888763750b4fc40R164).